engine/crates/federated-graph/src/render_sdl.rs

```diff
@@ -332,6 +323,33 @@ fn write_composed_directives(directives: Directives, graph: &FederatedGraphV2, s
                 write_quoted(sdl, &graph[*reason])?;
                 write!(sdl, ")")?;
             }
+            Directive::RequiresScopes(scopes) => {
+                write!(sdl, " @requiresScopes(scopes: [")?;
+                for scope in scopes {
+                    write!(sdl, "[")?;
+                    for scope in scope {
+                        write_quoted(sdl, &graph[*scope])?;
+                        write!(sdl, ", ")?;
+                    }
+                    write!(sdl, "], ")?;
+                }
+                write!(sdl, "])")?;
+            }
```

Comment:
This is not great, and doesn't look great when rendered, we should have a nice value printer.